CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# CONTRIBUTING
+
+Thanks, for your interest in contributing to Kubewarden.
+
+They are different ways you could help us simplify the adoption of policy as code.
+
+## Feedback
+
+* Starring our main repository [kubewarden/policy-server](https://github.com/kubewarden/policy-server). GitHub stars do matter !
+* Joining us on our [`kubewarden` channel](https://kubernetes.slack.com/?redir=%2Fmessages%2Fkubewarden).
+* Following us on [Twitter](https://twitter.com/kubewarden).
+
+## Code 
```

Comment:
I'm ok with this section, but it feels like an abstract of profile/README.md. I suppose one cannot reuse the profile part without a markdown preprocessor.

Reply:
> Thanks Olivier! I wonder what happens with clashing CONTRIBUTING.md files on other repos, for example
Nope, it's not possible at least as far as I know